about.html

```diff
@@ -19,21 +19,23 @@
             <a href="#" id="hamburguer-a" class="nav-link"><i class="fa-solid fa-bars fa-2xl"></i></a>
         </div>
         <nav id="top-navbar" class="container-fluid hidden">
-            <ul id="top-navbar-list" class="nav justify-content-end gap-2">
-                <li class="nav-item"><i class="fa-brands fa-facebook"></i></li>
-                <li class="nav-item">|</li>
-                <li class="nav-item"><i class="fa-brands fa-twitter"></i></li>
-                <li class="nav-item">|</li>
-                <li class="nav-item">English</li>
-                <li class="nav-item">|</li>
-                <li class="nav-item">My Page</li>
-                <li class="nav-item">|</li>
-                <li class="nav-item">Logout</li>
-            </ul>
+            <div class="top-nav-container">
+                <ul id="top-navbar-list" class="nav justify-content-end gap-2">
+                    <li class="nav-item"><i class="fa-brands fa-facebook"></i></li>
+                    <li class="nav-item">|</li>
+                    <li class="nav-item"><i class="fa-brands fa-twitter"></i></li>
+                    <li class="nav-item">|</li>
+                    <li class="nav-item">English</li>
+                    <li class="nav-item">|</li>
+                    <li class="nav-item">My Page</li>
+                    <li class="nav-item">|</li>
+                    <li class="nav-item">Logout</li>
+                </ul>
+            </div>
```

Comment:
- [Optional] consider using an indentation of two spaces in all your documents for readability. This is a convention and also [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).